ERROR - 2024-01-27 16:32:48 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 16:32:48 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`, `privilegios`.`privilegio`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id` = `ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 16:48:43 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 16:48:43 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 16:57:26 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 16:57:26 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 16:57:33 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 16:57:33 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:00:13 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:00:13 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:03:02 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:03:02 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:08:40 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:08:40 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:09:39 --> mysqli_sql_exception: Unknown column 'ingreso.id' in 'field list' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:09:39 --> Unknown column 'ingreso.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:10:54 --> mysqli_sql_exception: Unknown column 'ingreso.id' in 'field list' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:10:54 --> Unknown column 'ingreso.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id`, `ingreso`.`correo`, `ingreso`.`id_privilegio`
FROM `privilegios`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:12:56 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:12:56 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:12:58 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:12:58 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:14:04 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:14:04 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`, `privilegios`.`privilegio`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:14:05 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:14:05 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`, `privilegios`.`privilegio`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:16:49 --> mysqli_sql_exception: Unknown column 'ingreso.id' in 'field list' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:16:49 --> Unknown column 'ingreso.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`, `privilegios`.`privilegio`
FROM `privilegios`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:17:05 --> mysqli_sql_exception: Unknown column 'ingreso.id' in 'field list' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:17:05 --> Unknown column 'ingreso.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`
FROM `privilegios`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:18:16 --> mysqli_sql_exception: Unknown column 'ingreso.id' in 'field list' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:18:16 --> Unknown column 'ingreso.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`
FROM `privilegios`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:19:24 --> mysqli_sql_exception: Unknown column 'ingreso.id' in 'field list' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:19:24 --> Unknown column 'ingreso.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`
FROM `privilegios`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:19:36 --> mysqli_sql_exception: Unknown column 'id.ingreso' in 'field list' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`.`in...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`.`in...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`.`in...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `id`.`in...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:19:36 --> Unknown column 'id.ingreso' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `id`.`ingreso` as `clave`
FROM `privilegios`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:20:11 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id` as ...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id` as ...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id` as ...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `id` as ...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:20:11 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `id` as `clave`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:21:17 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:21:17 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `privilegio`.`id`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-27 17:23:49 --> mysqli_sql_exception: Not unique table/alias: 'privilegios' in C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `ingreso...', 0)
#1 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `ingreso...')
#2 C:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `ingreso...')
#3 C:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso...', Array, false)
#4 C:\xampp\htdocs\codeigniter\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter\app\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(942): App\Controllers\Usuarios->index()
#8 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\xampp\htdocs\codeigniter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-27 17:23:49 --> Not unique table/alias: 'privilegios'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `ingreso`.`id` as `clave`, `ingreso`.`correo`, `ingreso`.`id_privilegio`, `privilegios`.`id`, `privilegios`.`privilegio`
FROM `privilegios`
JOIN `privilegios` ON `privilegios`.`id`=`ingreso`.`id_privilegio`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-27 17:26:35 --> Undefined variable $data
in APPPATH\Controllers\Usuarios.php on line 27.
 1 APPPATH\Controllers\Usuarios.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'C:\\xampp\\htdocs\\codeigniter\\app\\Controllers\\Usuarios.php', 27)
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-27 17:34:04 --> Call to undefined method App\Models\IngresoModel::getCompiledSelected
in SYSTEMPATH\Model.php on line 873.
 1 APPPATH\Controllers\Usuarios.php(25): CodeIgniter\Model->__call('getCompiledSelected', [])
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-27 17:34:29 --> You cannot use "getCompiledSelect()" in "App\Models\IngresoModel". This is a method of the Query Builder Class.
in SYSTEMPATH\Model.php on line 889.
 1 SYSTEMPATH\Model.php(889): CodeIgniter\Exceptions\ModelException::forMethodNotAvailable('App\\Models\\IngresoModel', 'getCompiledSelect()')
 2 SYSTEMPATH\Model.php(869): CodeIgniter\Model->checkBuilderMethod('getCompiledSelect')
 3 APPPATH\Controllers\Usuarios.php(25): CodeIgniter\Model->__call('getCompiledSelect', [])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-27 17:38:31 --> Undefined variable $resultado
in APPPATH\Views\usuarios\listado.php on line 20.
 1 APPPATH\Views\usuarios\listado.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $resultado', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\listado.php', 20)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\listado.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/listado', [], true)
 5 APPPATH\Controllers\Usuarios.php(29): view('usuarios/listado', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-27 18:44:42 --> syntax error, unexpected token "}", expecting ";"
in APPPATH\Controllers\Usuarios.php on line 33.
 1 SYSTEMPATH\Autoloader\Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\codeigniter\\app\\Controllers\\Usuarios.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Usuarios')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Usuarios')
 4 SYSTEMPATH\CodeIgniter.php(900): class_exists('\\App\\Controllers\\Usuarios', true)
 5 SYSTEMPATH\CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 18:54:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 18:59:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 18:59:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:00:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:00:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:06:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:13:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:15:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:20:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:20:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:22:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:22:01 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:23:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:23:51 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:28:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:28:29 --> Undefined array key "privilegios"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegios"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:28:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:28:32 --> Undefined array key "privilegios"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegios"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:30:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:30:23 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:30:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:30:27 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:30:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:30:28 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:30:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:30:29 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:30:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:30:30 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:32:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:32:28 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:34:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:34:32 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:37:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:37:30 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:39:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:39:29 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:40:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:40:01 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:41:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:41:05 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:41:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:41:33 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:41:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:41:54 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:43:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-27 19:43:12 --> Undefined array key "privilegio"
in APPPATH\Views\usuarios\formulario.php on line 55.
 1 APPPATH\Views\usuarios\formulario.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "privilegio"', 'C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php', 55)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\codeigniter\\app\\Views\\usuarios\\formulario.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('usuarios/formulario', [], true)
 5 APPPATH\Controllers\Usuarios.php(33): view('usuarios/formulario', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuarios->alta()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-27 19:46:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:52:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:53:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:54:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-27 19:56:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
